nixos/modules/services/monitoring/riemann.nix

```diff
@@ -64,6 +93,12 @@ in {
           Extra Java options used when launching Riemann.
         '';
       };
+      environment = mkOption {
```

Comment:
Users can just set `systemd.services.riemann.environment` directly in their NixOS config, will work just as expected.

Reply:
I'm happy to take this out, just wasn't sure which was the more nixy way of doing things.

Reply:
the reason I ask this is because I am using this as:
```
      services.riemann = {
        enable = true;
        environment = { RIEMANN_HOME = riemannConfigDir; };
        configDirectory = riemannConfigDir;
      };
```
and it would not look so nice to me if the `environment` line was outside of the definition of `services.riemann`. What do you think @Infinisil @joachifm

Reply:
I think options for services should provide something more powerful than just an alias for another options. But I see your point, the config would be all neat and together.

Eventually I think a better solution to that is to provide a generic service option to be used to get to a services systemd service like this:
```nix
services.riemann = {
  enable = true;
  service.environment = { RIEMANN_HOME = riemannConfigDir; };
  configDirectory = riemannConfigDir;
}
```

This should probably then be a major nixpkgs change to implement it uniformly.

Reply:
yeah I like the sounds of that. However in the short term, do you want me to change this or leave it as is?

Reply:
I'd rather not have this